README.md

```diff
@@ -117,8 +117,22 @@ To load JDWP agent for UAA jvm debugging, start the server as follows:
 ```sh
 ./gradlew run -Dxdebug=true
 ```
+or
+```sh
+./gradlew -Dspring.profiles.active=default,hsqldb,debug run
+```
 You can then attach your debugger to port 5005 of the jvm process.
 
+To suspend the server start-up until the debugger is attached (useful for
+debugging start-up code), start the server as follows:
+```sh
```

Comment:
Just need one option here, too, if they both do the same thing.

Reply:
The same. I want to keep both options so one can also use the spring profile with all the desired options instead.